action.bash

```diff
@@ -64,7 +64,12 @@ case "$GITHUB_EVENT_NAME" in
 		else
 			TAG="${BASH_REMATCH[2]}"
 		fi
-		mkimg
+
+		if [ "${BASH_REMATCH[1]}" = heads ]; then
+			BRANCH="${BASH_REMATCH[2]}"
+		fi
```

Comment:
Can you please explain the reason for this change?

The whole case block is somewhat confusing and I'd find it much more helpful to document this block rather than what the regex does in detail.

Reply:
Aren't both the condition (is $REF a branch?) and the action (use that branch) obvious (w/o even knowing the array positions in advance)?